console/src/bindings/generators/templates/c.hbs

```diff
@@ -7,6 +7,6 @@ const struct _{{name}} {{name}} {
 };
 {{#each funcs}}
 LF_WEAK {{ret}} {{../name}}_{{name}}({{param_expansion params}}) {
-    {{ret}} result = lf_invoke(&_{{../name}}, _{{../name}}_{{name}}, {{ret_fmr}}, lf_args({{fmr_expansion params}}));
+    {{ret}} result = lf_invoke(lf_get_current_device(), &_{{../name}}, _{{../name}}_{{name}}, {{ret_fmr}}, lf_args({{fmr_expansion params}}));
```

Comment:
Just so you know, changing the generator code, specifically the templates, will necessarily cause all of the tests for the generator to break. I know we have more changes that need to be merged for these files so why don't we forego these changes in this PR to avoid breaking the build and then we'll just include these changes in the other PR and get that ready to merge right after this one?

Reply:
I need to update the rust bindings to reflect these changes, and I suppose that includes updating the tests. I would like to have this done by tonight if possible because it's blocking @harlanhaskins. Since it's just a parameter change to one function, I think the fix should be trivial.

Reply:
Don’t worry about blocking me, by the way.

Reply:
It needs to get done anyway. This gives me an impetus.